src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -197,7 +187,7 @@ private static double accumulate(Double a, Double b) {
               .thenAccept(lines::add));
           }
 
-          VertxCompletableFuture.allOf(ctx, futures.toArray(new CompletableFuture[futures.size()]))
+          VertxCompletableFuture.allOf(ctx, futures.toArray(new CompletableFuture[0]))
```

Comment:
I had to look into why this change was made... just learned something new!  

https://shipilev.net/blog/2016/arrays-wisdom-ancients/